action.yml

```diff
@@ -21,7 +21,7 @@ inputs:
     description: 'Comma separated list of Tags to be applied to nodes. The OAuth client must have permission to apply these tags.'
     required: false
   version:
-    description: 'Tailscale version to use.'
+    description: 'Tailscale version to use. Specify `latest` to use the latest stable version.'
```

Comment:
I'm not 100% sure where all this actually gets rendered.  Maybe add to the relevant section of the README file also?

Reply:
This gets automatically rendered by the actions marketplace. I've also updated the readme